shared/slices/slice.go

```diff
@@ -57,3 +57,13 @@ func Not(a []uint32, b []uint32) []uint32 {
 	}
 	return set
 }
+
+// IsIn returns true if a is in b and False otherwise.
+func IsIn(a uint32, b []uint32) bool {
```

Comment:
it's unfortunate these are purely for uint32's...in the future people might try to use this package thinking it is more powerful and can work for different types but eventually face a type mismatch. We can take care of this later, but important to keep in mind (same goes for my intersection function).

Reply:
Agreed. This is also an excellent first issue. I'll open one in a bit